test/index.test.ts

```diff
@@ -1,92 +1,125 @@
-import nock from 'nock'
-import approvemanApp from '../src'
-import { Probot } from 'probot'
-import prOpenedPayload from './fixtures/basic/pr.opened.json'
-import prReopenedPayload from './fixtures/basic/pr.reopened.json'
-import prSynchronizePayload from './fixtures/basic/pr.synchronize.json'
-import { setConfigToBasic, setConfigNotFound } from './utils/config'
-import { checkApproved, setSinglePreviousReview, verifyReviewDismissed } from './utils/review'
-const fs = require('fs')
-const path = require('path')
+import nock from "nock";
+import approvemanApp from "../src";
+import { Probot } from "probot";
+import prGeneric from "./fixtures/basic/pr.json";
+import { setConfigToBasic, setConfigNotFound } from "./utils/config";
+import { checkSuccessStatus } from "./utils/status";
+import {
+  checkApproved,
+  setSinglePreviousReview,
+  verifyReviewDismissed,
+} from "./utils/review";
+import fs from "fs";
+import path from "path";
+import { StatusCodes } from "http-status-codes";
+import { TEST_TIMEOUT } from "./utils/jest";
 
-jest.setTimeout(30000)
+jest.setTimeout(TEST_TIMEOUT);
 
-describe('Approveman tests', () => {
-  let probot: any
-  let mockCert: string
+const prOpenedPayload: any = prGeneric;
+prOpenedPayload.action = "opened";
+const prReopenedPayload: any = prGeneric;
+prOpenedPayload.action = "reopened";
+const prSynchronizePayload: any = prGeneric;
```

Comment:
Codacy found an issue: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)